frame/support/procedural/src/construct_runtime/parse.rs

```diff
@@ -471,7 +476,7 @@ impl PalletPartKeyword {
 
 	/// Returns the names of all pallet parts that allow to have a generic argument.
 	fn all_generic_arg() -> &'static [&'static str] {
-		&["Event", "Error", "Origin", "Config"]
+		&["Event", "Error", "Origin", "Config", "Task"]
```

Comment:
`Task` isn't required to be generic, yeah? If it is, then the syntax for including it is `Task<T>` in `construct_runtime`.